test/test-data-named-groups-unicode.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Out of curiosity: Where did you/@nicolo-ribaudo get these tests from? Are these the official ECMA-262 tests for the named-groups feature?

Reply:
No, I wrote them by myself.